client/src/components/ListsDisplay.js

```diff
@@ -65,7 +67,7 @@ const ListsDisplay = (props) => {
 
     useEffect(() => {
     getLists();
-  }, [userId]);
+  }, [listDelete]);
```

Comment:
I see what you did, but the better approach would have been for `const [lists, setLists] = useState([]);` to be in the context. Then instead of re-fetching when you delete a list, you could just remove the deleted list from the state directly